src/main/java/com/alexlasota/online_shop_product_service/controller/ProductController.java

```diff
@@ -0,0 +1,84 @@
+package com.alexlasota.online_shop_product_service.controller;
+
+import com.alexlasota.online_shop_product_service.dto.ProductAttributeDTO;
+import com.alexlasota.online_shop_product_service.dto.ProductDTO;
+import com.alexlasota.online_shop_product_service.exception.ProductNotFoundException;
+import com.alexlasota.online_shop_product_service.mapper.ProductMapper;
+import com.alexlasota.online_shop_product_service.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@RestController
```

Comment:
Adnotacja RestController pozwala nam na zwracanie Samych typów obiektów a nie ResponseEntity. W sytuacji kiedy mialbys dla powodzenia wiecej niz jeden status to mozesz uzyc responseEntity natomiast w innych przypadkach jak chcesz inny status niz default 200 to daj po prostu @ResponseStatus nad metoda z kontrolera i wybierz taki jaki chcesz